app/uk/gov/hmrc/gform/formtemplate/FormTemplateService.scala

```diff
@@ -60,11 +60,13 @@ class FormTemplateService(formTemplateRepo: Repo[FormTemplate], formTemplateRawR
         }
       })
 
-  def verifyAndSave(formTemplate: FormTemplate): FOpt[Unit] =
+  def verifyAndSave(formTemplate: FormTemplate)(expressionsContext: Substitutions): FOpt[Unit] = {
+    val substitutedFormTemplate = substituteExpressions(formTemplate, expressionsContext)
```

Comment:
If you import ```import uk.gov.hmrc.gform.formtemplate.Substituter.SubstituterSyntax```

this line can be

```
val substitutedFormTemplate = formTemplate(expressionsContext)
```

We may not need that trait `trait SubstituteExpressions`

Reply:
I don't want to make `SubstituterSyntax` widely available so I rather make it `private`

Reply:
Ok. It did cause a bit of confusion when I saw the usage for the first time in Substituter class. The IDE was not helping either